src/program/program.spec.ts

```diff
@@ -370,4 +386,38 @@ describe("Collection", () => {
 	test("should populate previous and next based on module ID", function () {
 		expect(true).toBe(true);
 	});
+	test("should previous and next based on module ID", function () {
+		expect(false).toBe(true);
+	});
+
+	describe("Creates", () => {
+		describe("Query.createcontent()", () => {
+			test("should create new content", async () => {
+				testingModuleCreateContentArgs = {
+					type: ContentType.PDF,
+					link: "test",
+					parent: fakelessonID,
+					primary: false
+				};
+				const createNewContent = await resolver.createContent(
+					testingModuleCreateContentArgs
+				);
+
+				if (createContent instanceof Error) throw new Error();
```

Comment:
- What are we checking for here?
- Please add a valid message to the error being thrown...
- Why are these changes here when the ticket was supposed to be for module objectives?